README.md

```diff
@@ -13,6 +13,15 @@
 **HTTPlug, the HTTP client abstraction for PHP.**
 
 
+## Intro
+
+HTTPlug served as a reference is the predecessor of [PSR-18](https://github.com/php-fig/fig-standards/blob/master/accepted/http-client/http-client.md)
+HTTP Client standard. Since there is an entire ecosystem built around HTTPlug and since it's already widely adapted,
+this package stays available, but new implementations and consumers should use the PSR-18 interfaces.
```

Comment:
could we say instead "we will keep maintaining this package for the time being, but new ..."